src/backend/src/controllers/users.controllers.ts

```diff
@@ -145,4 +145,14 @@ export default class UsersController {
       next(error);
     }
   }
+
+  static async getUserScheduleSettings(req: Request, res: Response, next: NextFunction) {
+    try {
+      const userId: number = parseInt(req.params.userId);
+      const userScheduleSettings = await UsersService.getUserScheduleSetting(userId);
```

Comment:
we should pass in the submitter and validate that it's the same as the id of the user who's schedule settings we're requesting. We should probably have a separate endpoint for retrieving availability given a set of userIds